lib/usage.js

```diff
@@ -261,10 +261,24 @@ module.exports = function (yargs, y18n) {
       })
 
       examples.forEach(function (example) {
-        ui.div(
-          {text: example[0], padding: [0, 2, 0, 2], width: maxWidth(examples, theWrap) + 4},
-          example[1]
-        )
+        if (example[1] === '') {
+          ui.div(
+            {
+              text: example[0],
+              padding: [0, 2, 0, 2]
```

Comment:
I think rather than having no wrap here, you should use `theWrap`; this way the example will still wrap onto multiple lines at the width boundary, but will take up a full line.